xml/System.IO/DirectoryInfo.xml

```diff
@@ -1577,38 +1577,6 @@
         <related type="Article" href="~/docs/standard/io/how-to-write-text-to-a-file.md">How to: Write Text to a File</related>
       </Docs>
     </Member>
-    <Member MemberName="FullName">
```

Comment:
not sure why this member is being deleted @joelmartinez

Reply:
Hmm, no this one is strange ... I fetched all the latest binaries and tried to reproduce locally, but was unable to (ie. this member remained in the xml repo). For now, just add this back (I assume you already have, cause it says outdated), and let's see what happens next time.